### DISCUSSION TITLE
Dedicated server from 3 years ago on the github.

### DISCUSSION THREAD

**Discussion by BlackMoons** [FakeFishGames/Barotrauma#14516] [Code questions and modding support]
https://github.com/FakeFishGames/Barotrauma/tree/master/Barotrauma/BarotraumaServer

Just wondering.. why is there a build of dedicatedserver.exe from 3 years ago here? lol.

**Comment by evilfactory** [comment]
It's not a build, not even an executable, it's just a bash file, go look at it.

**Reply by BlackMoons** [reply]
Hu.... Weird. Wonder why they gave it a .exe extension?
